scripts/create_manifest_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -e
```

Comment:
is `-e` actually what we want in this particular script, given that we do have handling through `die` et.al?